packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/CreateReceivables.tsx

```diff
@@ -164,7 +164,7 @@ const CreateReceivablesBase = ({
           quantity: item.quantity,
           product_id: item.product_id,
           ...(isNonVatSupported
-            ? { tax_rate_value: item?.tax_rate_value ?? 0 * 100 }
+            ? { tax_rate_value: item?.tax_rate_value * 100 ?? 0 * 100 }
```

Comment:
`item?.tax_rate_value` could be `undefined`, so the current one-line check should be refactored to `tax_rate_value: item?.tax_rate_value ? item?.tax_rate_value * 100 : 0 * 100`. There is also an improvement: 0 * 100 always produces 0, so it could be simplified to` tax_rate_value: item?.tax_rate_value ? item?.tax_rate_value * 100 : 0`